### PR TITLE
Add scss lint gem CASE 860

### PR DESCRIPTION
**Pull request by mlipardo**
Added a scss_lint gem. I think the `Add linting routines to yarn` needs to be merged first so I can resolve the conflicts.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -12,6 +12,8 @@ gem 'dry-types', '~> 0.12.2'
 gem 'faraday', '~> 0.13.1'
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
 gem 'rails', '~> 5.1.4'
+# Configurable tool for writing clean and consistent SCSS
+gem 'scss_lint', '~> 0.56.0', require: false
```
Comment (eckdanny-osi) [inline Gemfile]: ruby gem? :\

Can we use an npm dep (eg; `stylelint-cli` or something that integrates nicely with prettier)

**Comment by edgibbs** [comment]
So we're switching this to style lint with a prettier plugin?

**Comment by mlipardo** [comment]
Change from gem scss_lint to stylelint.

**Comment by mlipardo** [comment]
The PR #123 runs all the lint test, this would also run the lint:test:scss.

**Pull request merged**